### DISCUSSION TITLE
How to sync whole database? Please help me...

### DISCUSSION THREAD

**Discussion by uniorder** [apache/flink-cdc#2076] [Q&A]
In my case, I need to sync whole database that include thousands tables with real-time.

I can use  `insert into ... select ...` to sync one table, but if I have thousands table, Do I need to execute the same number of jobs?

That means I should be have much more servers?

I know this question is not about flink-cdc-connector, but I really want to know the answer, thank you every body.

**Comment by hbk671104** [comment]
You might wanna try the `DataStream` version of cdc connectors, where most of them provide database / multi-table capture support.

- [Oracle DataStream Connector](https://github.com/ververica/flink-cdc-connectors/blob/master/docs/content/connectors/oracle-cdc.md#datastream-source)
- [MySQL DataStream Connector](https://github.com/ververica/flink-cdc-connectors/blob/master/docs/content/connectors/mysql-cdc.md#datastream-source)
